<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenuClass</class>
 <widget class="QWidget" name="MainMenuClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>598</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>200</horstretch>
    <verstretch>200</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Triviador</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource/Triviador.qrc">
    <normaloff>:/icon/Triviador.ico</normaloff>:/icon/Triviador.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(83, 66, 50);
color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,10,10,10,10,0">
     <item>
      <widget class="QLabel" name="icon">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resource/Triviador.qrc">:/icon/Triviador-castle.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="logOutButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Franklin Gothic Heavy</family>
         <pointsize>44</pointsize>
         <underline>false</underline>
         <strikeout>false</strikeout>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background: rgb(128, 18, 18);
border-radius: 30px;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resource/Triviador.qrc">
         <normaloff>:/icon/Logout.png</normaloff>:/icon/Logout.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="hiMessage">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="playButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Franklin Gothic Heavy</family>
         <pointsize>35</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(62, 45, 30);
border: none;</string>
       </property>
       <property name="text">
        <string>PLAY</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="myProfileButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Franklin Gothic Heavy</family>
         <pointsize>35</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(62, 45, 30);
border: none;</string>
       </property>
       <property name="text">
        <string>MY PROFILE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="creditsButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Franklin Gothic Heavy</family>
         <pointsize>35</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(62, 45, 30);
border: none;</string>
       </property>
       <property name="text">
        <string>CREDITS</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>200</horstretch>
       <verstretch>200</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="play">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <widget class="QPushButton" name="joinGameButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Franklin Gothic Heavy</family>
             <pointsize>44</pointsize>
             <underline>false</underline>
             <strikeout>false</strikeout>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(128, 18, 18);
border: none;
</string>
           </property>
           <property name="text">
            <string>JOIN GAME</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="createGameButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Franklin Gothic Heavy</family>
             <pointsize>44</pointsize>
             <underline>false</underline>
             <strikeout>false</strikeout>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(128, 18, 18);
border: none;
</string>
           </property>
           <property name="text">
            <string>CREATE GAME</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="joinGame">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <spacer name="verticalSpacer_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="1">
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Franklin Gothic Heavy</family>
             <pointsize>35</pointsize>
            </font>
           </property>
           <property name="text">
            <string>ENTER THE GAME ID:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="joinLobbyButton">
           <property name="font">
            <font>
             <family>Franklin Gothic Heavy</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background: rgb(128, 18, 18);
</string>
           </property>
           <property name="text">
            <string>JOIN</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer_11">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="createGame">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="3">
          <widget class="QLabel" name="modeLabel_2">
           <property name="font">
            <font>
             <family>Franklin Gothic Heavy</family>
             <pointsize>25</pointsize>
            </font>
           </property>
           <property name="text">
            <string>CREATE GAME</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="twoPlayersButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Franklin Gothic Heavy</family>
             <pointsize>25</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background: rgb(128, 18, 18);
border: none;</string>
           </property>
           <property name="text">
            <string>2 PLAYERS</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="backButton">
           <property name="font">
            <font>
             <family>Franklin Gothic Heavy</family>
             <pointsize>44</pointsize>
             <underline>false</underline>
             <strikeout>false</strikeout>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background: rgb(128, 18, 18);
border-radius: 30px;
</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../resource/Triviador.qrc">
             <normaloff>:/icon/back_arrow.png</normaloff>:/icon/back_arrow.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>100</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QPushButton" name="createButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Franklin Gothic Heavy</family>
             <pointsize>29</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background: rgb(128, 18, 18);
border: none;</string>
           </property>
           <property name="text">
            <string>CREATE</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QPushButton" name="threePlayersButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Franklin Gothic Heavy</family>
             <pointsize>25</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background: rgb(128, 18, 18);
border: none;</string>
           </property>
           <property name="text">
            <string>3 PLAYERS</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="modeLabel">
           <property name="font">
            <font>
             <family>Franklin Gothic Heavy</family>
             <pointsize>25</pointsize>
            </font>
           </property>
           <property name="text">
            <string>MODE:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QPushButton" name="fourPlayersButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Franklin Gothic Heavy</family>
             <pointsize>25</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background: rgb(128, 18, 18);
border: none;</string>
           </property>
           <property name="text">
            <string>4 PLAYERS</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="5">
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="myProfile">
      <widget class="QLabel" name="profileInformation">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>2353</width>
         <height>176</height>
        </rect>
       </property>
       <property name="text">
        <string>Roșia (pătlăgica roșie sau mai rar tomata), numită științific Solanum lycopersicum, este o plantă din familia Solanaceae, apropiată pe linie genetică de următoarele plante, de asemenea originare din &quot;Lumea Nouă&quot;: tutun, ardei, cartof, vânătă și physalis alkekengi. Roșia este o plantă nativă a sudului Americii de Nord și nordului Americii de Sud, având un areal natural de extindere din centrul Mexicului până în Peru.

Este o plantă perenă, crescută adesea în climate temperate ca o plantă anuală, atingând frecvent între 1 și 3 m înălțime, cu un trunchi mai mult ierbos, care crește de cele mai multe ori, ca orice altă plantă cățărătoare, pe tulpinile altor plante, pe bețe sau garduri.

Este cultivată la aproape toate latitudinile, ocupând o suprafață de aproximativ 3 milioane de hectare, adică o treime din terenul consacrat legumelor.[necesită citare]

Frunzele plantei sunt lungi de 10–25 cm, de formă penat-compusă, având 5–9 frunzulițe, fiecare dintre acestea având până la 8 cm lungime, cu margini serate. Atât trunchiul cât și crenguțele, respectiv frunzele sunt acoperite cu excrescențe asemănătoare părului animalelor. Florile sunt mici, de circa 1–2 cm, galbene, având corola formată din cinci sepale și cinci petale, fiind grupate în inflorescențe de 3 până la 12 flori. Rosia are un pH acid de 5,5.

Cuvântul tomată, respectiv toate variantele sale din spaniolă, engleză și alte limbi, este un cuvânt derivat din limba populațiilor mezo-americane Nahua: tomatl. Numele științific al plantei, care provine din latină, lycopersicum, semnifică &quot;piersică-lup&quot;, conform lyco - lup și persicum - piersic(ă).

Din punct de vedere botanic roșia este un fruct.[1] În anul 1893 Curtea Supremă a Statelor Unite ale Americii în mod unanim a decis că la colectarea taxelor vamale roșiile trebuie considerate legume (deși curtea a trecut în revistă faptul că din punct de vedere botanic, roșiile sunt fructe).[2] În anul 2001 Uniunea Europeană a decis oficial că roșia este fruct, și nu legumă.[3]</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="credits"/>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resource/Triviador.qrc"/>
 </resources>
 <connections/>
</ui>
